Extensions/Leaderboards/leaderboardstools.ts

```diff
@@ -7,10 +7,36 @@ namespace gdjs {
       let _lastPlayerName: string;
       let _lastErrorCode: number;
       let _leaderboardViewIframe: HTMLIFrameElement | null = null;
+      let _leaderboardViewIframeErrored: boolean = false;
+      let _leaderboardViewIframeLoading: boolean = false;
+      let _leaderboardViewIframeLoaded: boolean = false;
+      let _errorTimeoutId: NodeJS.Timeout | null = null;
```

Comment:
Interesting, why the choice of `NodeJS.Timeout`?

Reply:
That's the type returned by `setTimeout` 😅